### PR TITLE
RIUI-314 Reinstate questions check page

### PR DESCRIPTION
**Pull request by Louisblack**
Also un-ignoring the questions-panel tests from last week.

### CODE REVIEW

File: api/questions/question.js [hunk 1]
```diff
@@ -54,7 +54,10 @@ function formatQuestionRes(question, answers) {
 
 function getQuestionsByCase(caseId, userId, options, jurisdiction) {
     return getHearingByCase(caseId, options)
-        .then(hearing => hearing.online_hearings[0] ? hearing.online_hearings[0].online_hearing_id : postHearing(caseId, userId, options, jurisdiction))
+        .then(hearing => {
+            console.log(JSON.stringify(hearing));
```
Comment (alectronic0) [inline api/questions/question.js]: should we be logging this?

Comment (Louisblack) [reply api/questions/question.js]: You're right. I left it in there for debugging. I'll update it

File: api/questions/question.js [hunk 2]
```diff
@@ -127,6 +130,27 @@ module.exports = (app) => {
             });
     });
 
+    route.get('/:case_id/questions', (req, res, next) => {
```
Comment (alectronic0) [inline api/questions/question.js]: has this been moved from somewhere else?

Comment (Louisblack) [reply api/questions/question.js]: Nope. We didn't have an end point for getting questions. We are using the `getQuestionsByCaseId` function to add them to the case file.

File: api/questions/question.js (hunk 1 above)
Comment (alectronic0) [reply api/questions/question.js]: just getting you back for all the time you do it to me :stuck_out_tongue_winking_eye:

Comment (Louisblack) [reply api/questions/question.js]: lol. Nah, it's cool. I've removed it.

**Pull request merged**